_pages/en_US/Uninstall-Mocha-CFW.txt

```diff
@@ -39,10 +39,11 @@ Games and saves are saved to the internal memory not the SD card. This means all
 ##### Section II - Removal from SD Card
 
 {% capture notice-3 %}
-**Remove any extra files and folders from the root of the SD card that are *not* in the following list (you may not have all or even any of these):**
+**You can safely remove these files and folders from the root of the SD card in the following list if you don't intend to reinstall Mocha/Haxchi:**
 
-    + DCIM
-    + private
+    + /wiiu/apps/haxchi
+    + /wiiu/apps/mocha
+    + /haxchi
```

Comment:
Why do you keep pull requesting this change? This is for reverting to stock and there are other folders that need removing...